solarized.el

```diff
@@ -52,6 +52,13 @@ Related discussion: https://github.com/bbatsov/solarized-emacs/issues/158"
   :type 'boolean
   :group 'solarized)
 
+(defcustom solarized-highlight-numbers nil
```

Comment:
I'd probably name this `solarized-font-lock-numbers`, so it's more in line with Emacs conventions.